src/index.css

```diff
@@ -1,10 +1,8 @@
-body {
-  margin: 0;
-  font-family: "Roboto", -apple-system, BlinkMacSystemFont, "Segoe UI", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
 
-code {
-  font-family: "Courier New", source-code-pro, Menlo, Monaco, Consolas, monospace;
+body,
+html {
+  font-family: 'Montserrat', sans-serif !important;
```

Comment:
* Kindly follow the [design guidelines](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=You%20should%20follow,space%20between%20elements) by using the right font family ( `gill sans`, `Lato` )

Reply:
Thanks for the review.
I have change the font family to lato